pyproject.toml

```diff
@@ -22,7 +22,7 @@ convention = "google"
 max-complexity = 10
 
 [tool.mypy]
-python_version = "3.9"
+python_version = "3.12"
```

Comment:
Mypy should do checks based on the oldest supported version of python. So 3.11 in this case?

Reply:
I didn’t know that, fixed, thanks.

Reply:
ah, now mypy is complaining again…

Reply:
Maybe just add type ignores for those lines. It is either numpy or the typeshed type definitions for the standard library's hash stuff that doesn't understand that numpy arrays support the buffer interface.

Reply:
Fixed, hopefully